### PR TITLE
Fix disable adaper test

### PR DESCRIPTION
**Pull request by younesbelkada**
Addresses: https://github.com/huggingface/peft/pull/683#issuecomment-1632253761

TLDR of the PR (in addition to what discussed internally):

1. I don't think we should check for `merge_adapter` method
2. We can test with merged model but the test should be adapted accordingly, in fact, we should check that the logits are completely off as the model has been merged with random adapter weights

What do you think ? 

the adaption prompts test are still failing, will investigate more, but meanwhile if you feel so you can merge the PR

### CODE REVIEW

**Comment by younesbelkada** [comment]
WIll propose a better fix that also checks for `merge_adapter` method

**Comment by BenjaminBossan** [comment]
Thanks for investigating!
